README.md

```diff
@@ -19,7 +19,7 @@ Code manager: Jeff Ator
 <pre>
 git clone https://github.com/noaa-emc/nceplibs-bufr
 mkdir build && cd build
-cmake -DCMAKE_INSTALL_PREFIX=./install -DENABLE_TESTS=ON nceplibs-bufr
+cmake -DCMAKE_INSTALL_PREFIX=./install -DENABLE_TESTS=ON ../nceplibs-bufr
```

Comment:
This should be `./nceplibs-bufr`

Reply:
No, it's ../nceplibs-bufr because the prior command (mkdir build && cd build) has now put you into a different subdirectory, at the same level as the nceplibs-bufr subdirectory.